functions/src/shared/delete-collection.ts

```diff
@@ -0,0 +1,44 @@
+// Dashboard hub firebase functions models/mappers
+import { Firestore, Query, QueryDocumentSnapshot, QuerySnapshot, WriteBatch } from '../client/firebase-admin';
+
+/**
+ * From https://firebase.google.com/docs/firestore/manage-data/delete-data
+ */
+export function deleteCollection(db: Firestore, collectionPath: string, batchSize: number): Promise<any> {
```

Comment:
It doesn't look like this is used anywhere in the project? I know this was in another PR but it is in this PR now

Reply:
It might be because of a bad merge conflict. I think it is better to resolve the conflicts the correct way around, and re-open the other PR, this way we can review, resolve and merge smaller changes